src/domains/wallet-app/api/webview-payment-wallet/v1/_openapi.json.tpl

```diff
@@ -0,0 +1,381 @@
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Wallet not found",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal Server Error"
+          },
+          "504": {
+            "description": "Timeout serving request"
+          }
+        }
+      }
+    },
+    "/wallets/{walletId}/outcomes": {
+      "get": {
+        "tags": [
+          "payment-wallet-webview"
+        ],
+        "summary": "Get wallet onboarding result",
+        "description": "Get wallet onboarding result and walletId as query params to return ii to the app IO, for example \n/wallets/{walletId}/outcomes?outcome=0&walletId=123. The possible outcome are:\n- SUCCESS(0)\n- GENERIC_ERROR(1)\n- AUTH_ERROR(2)\n- TIMEOUT(4)\n- CANCELED_BY_USER(8)\n- INVALID_SESSION(14)",
+        "operationId": "getWalletOutcomesById",
+        "security": [
+          {
+            "bearerAuth": []
+          }
+        ],
+        "parameters": [
+          {
+            "name": "walletId",
+            "in": "path",
+            "description": "ID of wallet",
+            "required": true,
+            "schema": {
+              "$ref": "#/components/schemas/WalletId"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Form fields retrieved successfully",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/WalletFieldsResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid input id",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Wallet not found",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal Server Error"
+          },
+          "504": {
+            "description": "Timeout serving request"
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "WalletId": {
+        "description": "Wallet identifier",
+        "type": "string",
+        "format": "uuid"
+      },
+      "WalletVerifyRequestsResponse": {
+        "type": "object",
+        "description": "Data to perform a new wallet verify requests",
```

Comment:
Are you sure this is the right description. It seems that this is a description for the reqeust